tutorhastexo/patches/k8s-deployments

```diff
@@ -1,3 +1,4 @@
+{% if HASTEXO_ENABLE_SUSPENDER %}
```

Comment:
Unless I'm gravely mistaken, this won't work. With this approach, if I flip `HASTEXO_ENABLE_SUSPENDER` to false on an existing system, Tutor will simply remove this bit from the configuration, but this will mean that the Deployment remains untouched (and will continue to run).

Hence the earlier suggestion of always enabling the Deployment, but "scale" it to 0 replicas if this boolean is false.

Reply:
Ack, fixed now.